.github/workflows/pull_request.yml

```diff
@@ -5,15 +5,12 @@ on:
     branches:
       - main
       - 7.[0-9]+.x
-    paths-ignore:
-      - 'optaplanner-8-benchmarks/**'
 
   pull_request:
     branches:
       - main
       - 7.[0-9]+.x
     paths-ignore:
-      - 'optaplanner-8-benchmarks/**'
```

Comment:
Also Java 8 can be removed from matrix and possibly be changed for Java 17 for example.